pkg/control/server.go

```diff
@@ -163,6 +167,26 @@ func NewServer(cfg ServerConfig) (*Server, error) {
 		return nil, err
 	}
 
+	var (
+		hubImageTag  string
+		hubImageFile string
+	)
+
+	if cfg.HubImageTag != "" && cfg.HubImageTag[0] == '@' {
+		L.Info("fetected hub image tag is a file, monitoring it for changes")
+
+		hubImageFile = cfg.HubImageTag[1:]
+
+		data, err := ioutil.ReadFile(hubImageFile)
+		if err != nil {
+			return nil, err
+		}
+
+		hubImageTag = string(bytes.TrimSpace(data))
+	} else {
+		hubImageTag = cfg.HubImageTag
```

Comment:
I guess we don't care that this could set it to an empty string?

Reply:
Yup, if it's empty string we do the same behavior, which is just pass on the empty string.